jlibra-examples/src/main/resources/log4j2.properties

```diff
@@ -4,7 +4,11 @@ appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
 appender.console.layout.pattern = %d{HH:mm:ss.SSS}| %m%n
 appender.console.filter.threshold.type = ThresholdFilter
-appender.console.filter.threshold.level = info
```

Comment:
Does this work with log4j/log4j2 properties?

Reply:
I'm not sure I understand, are you referring to a line in the file or the configuration overall? I did test this and the log contained everything it had before + the lines coming now from the core module.